shell/browser/api/electron_api_app.cc

```diff
@@ -1833,6 +1860,14 @@ gin::ObjectTemplateBuilder App::GetObjectTemplateBuilder(v8::Isolate* isolate) {
       .SetProperty("userAgentFallback", &App::GetUserAgentFallback,
                    &App::SetUserAgentFallback)
       .SetMethod("configureHostResolver", &ConfigureHostResolver)
+#if BUILDFLAG(IS_WIN)
+      .SetProperty("notificationsComServerCLSID",
+                   &App::GetBrowserClientNotificationsComServerCLSID,
+                   &App::SetBrowserClientNotificationsComServerCLSID)
+      .SetProperty("notificationsComDisplayName",
```

Comment:
This property does not appear to be documented

Reply:
Ok, agree looks like I've forgot to add description for next properties